### PR TITLE
Add more Blinka Info with photos swapped

### PR DESCRIPTION
**Pull request by makermelissa**
This PR supersedes #577.

### CODE REVIEW

**Comment by makermelissa** [comment]
@ptorrone fyi, this is just waiting on a review, but is otherwise complete.

File: blinka.html [hunk 1]
```diff
@@ -6,6 +6,9 @@
 
 <div id="downloads-page">
   <h1>Blinka</h1>
+  <p>
+    Blinka brings CircuitPython APIs and, therefore, libraries to single board computers (SBCs). It is a <a href="https://pypi.org/project/Adafruit-Blinka/">pip installable Python library</a> that runs in normal "desktop" Python. The CircuitPython runtime isn't used. <a href="https://pypi.org/search/?q=adafruit-circuitpython">Libraries</a> can also be installed via pip. See <a href="https://learn.adafruit.com/circuitpython-on-raspberrypi-linux"> the guide</a> for further details.
```
Comment (kattni) [inline blinka.html:10]: Consider including `CircuitPython` before libraries. I had to read the sentence a couple of times, and that would clarify things. Including "CircuitPython" in the later "Libraries" sentence might make sense as well. Up to you though! 
```suggestion
    Blinka brings CircuitPython APIs and, therefore, CircuitPython libraries to single board computers (SBCs). It is a <a href="https://pypi.org/project/Adafruit-Blinka/">pip installable Python library</a> that runs in normal "desktop" Python. The CircuitPython runtime isn't used. <a href="https://pypi.org/search/?q=adafruit-circuitpython">CircuitPython libraries</a> can also be installed via pip. See <a href="https://learn.adafruit.com/circuitpython-on-raspberrypi-linux"> the guide</a> for further details.
```

**Review by kattni — CHANGES_REQUESTED** [review]
@makermelissa Thank you for adding this! Potential suggestion below. If you'd rather not, let me know and I'll approve this.

**Comment by makermelissa** [comment]
Thanks @kattni, it sounds good to me.

**Review by kattni — APPROVED** [review]

**Pull request merged**